### ISSUE TITLE
Model trained on training+validation set

### ISSUE THREAD

**Issue by yankee624** [showlab/videollm-online#30]
According to the readme, the [released model](https://huggingface.co/chenjoya/videollm-online-8b-v1plus) is trained on Ego4D Narration Stream 113K + Ego4D GoalStep Stream 21K.
I guess it is trained with this script: [`scripts/ego4d/live1+.sh`](https://github.com/showlab/videollm-online/blob/main/scripts/ego4d/live1%2B.sh), which uses `ego4d_goalstep_livechat_trainval`, `ego4d_refined_narration_stream_train`, `ego4d_refined_narration_stream_val`, `robustness` to train the model.

But in Table1 of the paper, the model here seems to be trained only on `ego4d_refined_narration_stream_train`. Is this correct?
If so, can you please release this model? 
The reason I'm requesting this is because: the currently released model is trained on `ego4d_refined_narration_stream_train` + `ego4d_refined_narration_stream_val`, so there is no remaining validation/test set I can use to test model performance. If there is other test set I can use, please let me know.
